rxandroid-framework/src/main/java/rx/android/app/support/ReactiveDialog.java

```diff
@@ -0,0 +1,175 @@
+package rx.android.app.support;
+
+import android.content.DialogInterface;
+import android.os.Bundle;
+import android.support.v4.app.FragmentManager;
+import rx.Observable;
+import rx.Observer;
+import rx.Subscriber;
+import rx.Subscription;
+import rx.android.app.SubscriberVault;
+import rx.functions.Func1;
+
+/**
+ * Wrapper for DialogFragment that allows to observe on the result of the user interaction.
+ *
+ * @param <T> The type of data expected as return value from the fragment, can be boolean for confirmation dialogs,
+ *            or more complex for data input dialogs.
+ */
+public class ReactiveDialog<T> extends RxDialogFragment {
```

Comment:
This is almost straight-up copy/pasted from ReactiveDialog, and it shares the same name.

The code can definitely be consolidated.